integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/call/CallMediatorBlockingDirectEndpointTestCase.java

```diff
@@ -48,6 +52,17 @@ public void callMediatorBlockingDirectEndpointTest() throws AxisFault {
         assertTrue(responseContainsWSO2);
     }
 
+    @Test(groups = { "wso2.esb" },
+          description = "Invoke a backend that returns a 204, empty response")
+    public void callMediatorBlockingNoContentResponseTest() throws IOException {
+        SimpleHttpClient client = new SimpleHttpClient();
+        Map headers = new HashMap(1);
+        headers.put("Content-Type", "application/json");
+        org.apache.http.HttpResponse response = client.doGet("http://localhost:8480/CallMediatorNoContentTestAPI/",
```

Comment:
Can't we use getApiInvocationURL() method without hard coding the host and port

Reply:
Fixed. Thanks.